packages/joint-core/src/dia/Element.mjs

```diff
@@ -357,9 +360,18 @@ export const Element = Cell.extend({
         }
 
         // Set new size and position of this element, based on:
-        // - union of bboxes of all children
+        // - union of bboxes of filtered element children
         // - inflated by given `opt.padding`
+        // - containing at least `opt.minRect`
         this._fitToElements(Object.assign({ elements: childElements }, opt));
+        if (opt.minRect) {
```

Comment:
why is this logic here (and duplicated) and no in the `_fitToElements()` function?

The function also currently sets `position` and `size` twice.